### PR TITLE
Feature/milestone 02

### PR DESCRIPTION
**Pull request by carlosveigadev**
- Add main files and separate the components folder;
- Add App, Display, ButtonPanel, and Button content;
- Add Default values for some elements;
- Add type requirements for props.

### CODE REVIEW

File: src/components/App.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import Display from './Display';
+import ButtonPanel from './ButtonPanel';
+
+const App = () => (
+  <div>
+    <Display />
+    <ButtonPanel />
+  </div>
+);
```
Comment (wathigo) [inline src/components/App.js]: Has children components wrapped in React.Fragment short syntax ([screenshot from Pathwright](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone2/app/ReactFragment.png))

**Review by wathigo — CHANGES_REQUESTED** [review]
## CHANGES REQUIRED :o: 
Great job on this project.
Let's fix one issue before we proceed to the next milestone. see the comment :point_down: 

**After making the changes, submit another request for review.**

**Happy Coding :computer: 
Simon Wathigo**

**Comment by carlosveigadev** [comment]
Hi @wathigo, thanks for the review.
I implemented the improvements as suggested.

**Review by trekab — APPROVED** [review]
#### Approved! :tada: 
Well done @carlosveigadev! You may proceed to your next milestone.

__Treasure Kabareebe__
[Portfolio](https://trekab.github.io/portfolio/) | [LinkedIn](https://www.linkedin.com/in/treasure-kabareebe/) | [Twitter](https://twitter.com/TKabareebe)

**Pull request merged**